addons/account/static/src/services/account_notification_service.js

```diff
@@ -0,0 +1,32 @@
+/** @odoo-module **/
+
+import { _t } from "@web/core/l10n/translation";
+import { registry } from "@web/core/registry";
+
+
+export const accountNotificationService = {
+    dependencies: ["bus_service", "notification", "action"],
+
+    start(env, { bus_service, notification, action }) {
+        bus_service.subscribe("account_notification", ({ message, sticky, title, type, action_button}) => {
+            const buttons = [{
+                name: action_button.name,
+                primary: false,
+                onClick: () => {
+                    action.doAction({
+                        name: _t(action_button.action_name),
+                        type: 'ir.actions.act_window',
+                        res_model: action_button.model,
+                        domain: [["id", "in", (action_button.res_ids)]],
+                        views: [[false, 'list'],[false, 'form']],
+                        target: 'current',
+                    });
+                },
+            }];
+            notification.add(message, { sticky, title, type, buttons });
+        });
+        bus_service.start();
```

Comment:
Are you sure you need to start this service yourself? :thinking:

Reply:
Looks like no :thinking: 
I used another implementation of such notification as example that had it, but some others don't... most likely it just need to be started once for the instance.